programs/programs/policyengine/engines.py

```
@@ -58,39 +58,49 @@ def members(self, unit, sub_unit):
+        "audience": "https://household.api.policyengine.org",
+    }
+    try:
+        res = requests.post(_pe_token_url, json=payload, timeout=(5, 30))
+        res.raise_for_status()
+        data = res.json()
+        token = data["access_token"]
+        expires_in = int(data.get("expires_in", 86400))
+    except requests.RequestException as e:
+        raise RuntimeError(f"Failed to fetch PolicyEngine bearer token: {e}") from e
+    except (ValueError, KeyError) as e:
+        raise RuntimeError(f"Invalid response from PolicyEngine token")
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)